packages/completer/src/tokens.ts

```diff
@@ -104,10 +115,12 @@ export interface ICompletionProvider<
    * @param  completerIsVisible - Current visibility status of the
    *  completer widget0
```

Comment:
```suggestion
   *  completer widget
```

While we are at it..